experiment_config/bidaf.json

```diff
@@ -77,13 +82,12 @@
     "batch_size": 40
   },
   "optimizer": {
-    "type": "adadelta",
-    "lr": 0.5,
-    "rho": 0.95
+    "type": "sgd",
+    "lr": 0.00001
   },
   "num_epochs": 12,
-  "grad_norm": 100.0,
-  "grad_clipping": 100,
+  "grad_norm": 0.1,
```

Comment:
I was just playing around with a lot of different values, trying to get something to actually train.  I just removed it entirely, just using gradient clipping, and not normalization, and loss is finally going down.  I think there are too many parameters to have this norm make sense.